src/test/java/catan/controllers/user/UserControllerTest.java

```diff
@@ -8,20 +11,24 @@
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import static com.jayway.restassured.RestAssured.given;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.isEmptyString;
+import static org.hamcrest.Matchers.*;
 
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringApplicationConfiguration(classes = ApplicationConfig.class)
 @WebIntegrationTest("server.port:8091")
-public class UserControllerTest {
+public class UserControllerTest extends FunctionalTestUtil {
```

Comment:
ah, didn't see anything related to 'details' method in your TODO comments, thats why added this comment, but if you meant that everythig should be replaced, than ok)